composer.json

```
@@ -8,7 +8,7 @@
     "require": {
         "symfony/console": "^4 || ^5 || ^6",
         "symfony/process": "^4 || ^5 || ^6",
-        "drush/drush": "^10 || ^11 || ^12",
+        "drush/drush": "^10 || ^11 || ^12 || ^13",
```

Since there are no problems with so many further releases, let's use a more generic solution:

```suggestion
        "drush/drush": ">=10",
```